Add drive invitation on readme (#245)
owncloud/ocis-php-sdk: README.md

$resources = $drives[0]->getResources("/documents");
```

### Drive Permission

> Users/Groups can be invited to drives by specifying permissions. Drive class has method to update permission roles and expiration dates, remove users and groups, etc. related to drive invitations.

Reviewer: ```suggestion
Users/Groups can be invited to drives by specifying permissions. The `Drive` class has methods to invite Users/Groups, update permission roles and expiration dates, remove users and groups, etc.
```

> Drive invitations are allowed only on the **project drive**. For drive invitation, **root endpoint** has been used.

Reviewer: ```suggestion
Drive invitations are only possible on **project drives**.
```

I don't think the user of the SDK needs to know what endpoint is used

```php
// find all users with a specific surname
$users = $ocis->getUsers("einstein")[0];

// get all drives of type project
$drives = $ocis->getMyDrives(
    DriveOrder::NAME,
    OrderDirection::ASC,
    DriveType::PROJECT
);

> // get project drive name game

Reviewer: ```suggestion
// get the drive named 'game'
```

foreach ($drives as $drive) {
    if ($drive->getName) === 'game' {
        $gameDrive = $drive;
        break;
    }
}

// get all roles that are possible for that drive
$driveRoles = $gameDrive->getRoles();

> // get role that allowed to view, download, upload, edit, add, delete and manage members

Reviewer: ```suggestion
// get the role that is allowed to view, download, upload, edit, add, delete and manage members
```
